src/Components/Server/src/Circuits/IJSRuntimeAccessor.cs

```diff
@@ -1,10 +1,12 @@
 // Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
+using System;
 using Microsoft.JSInterop;
 
 namespace Microsoft.AspNetCore.Components.Server.Circuits
 {
+    [Obsolete("Resolve the current JavaScript runtime from the dependency injection container instead.")]
```

Comment:
Do we need all of this in this PR, would it be straightforward to remove this? (do we need an issue)

I agree that we should get rid of these things if we can.

Reply:
It would be, this is just garbage left behind from the original attempt. I can either revert these changes or remove the accessor completely. I believe is like 5/10 lines of code changes total. I'm happy to do so in another Pr too

Reply:
If this is no longer an API we have any use case for, I'd suggest dropping it now rather than marking obsolete. Better to make breaking changes sooner than later.

Reply:
Got rid of it.